js/Ball.js

```diff
@@ -5,14 +5,45 @@ class Ball {
         this.ballRadius = 10; // promień piłeczki
     }
 
-    drawBall(){
+    drawBall(ballX, ballY){
         // TO DO -> metoda tworząca piłeczkę (metody canvas'a)
-        // hint: jako argument funkcja przyjmuję x i y zdefiniowane globalnie w index.js 
+        // hint: jako argument funkcja przyjmuję x i y zdefiniowane globalnie w index.js
+        this.ctx.beginPath();
+        this.ctx.arc(ballX,ballY,this.ballRadius,0,2*Math.PI);
+        this.ctx.stroke();
     };
 
-    bouncing(){
+    bouncing(ballX, ballY, paddle){
```

Comment:
aż się prosi żeby sobie zdestrukturyzować to paddle
bouncing(ballX, ballY. { paddleX, paddleWidth })